packages/validator/src/services/attestation.ts

```diff
@@ -92,14 +92,16 @@ export class AttestationService {
       await Promise.all(
         Array.from(dutiesByCommitteeIndex.entries()).map(([index, duties]) =>
           this.runAttestationTasksPerCommittee(duties, slot, index, signal).catch((e) => {
-            this.logger.error("Error on attestation routine", {slot, index}, e);
+            this.logger.error("Error on committee attestation routine", {slot, index}, e);
           })
         )
       );
     } else {
       // Beacon node's endpoint produceAttestationData return data is not dependant on committeeIndex.
       // Produce a single attestation for all committees and submit unaggregated attestations in one go.
-      await this.runAttestationTasksGrouped(duties, slot, signal);
+      await this.runAttestationTasksGrouped(duties, slot, signal).catch((e) => {
+        this.logger.error("Error on attestation routine", {slot}, e);
+      });
```

Comment:
using both `await` and `catch` is a bit confused. If we don't want to wait for the result of `runAttestationTasksGrouped` we should remove `await` and we can use `catch` there, if we want to wait and catch the error, we should use `try/catch` pattern

Reply:
Let's use try-catch then, waiting for the result is important right now due to `runAtMostEvery`, this might change in the future as discussed here https://github.com/ChainSafe/lodestar/issues/5314. But unless there is a specific use case should probably prefer awaiting results in most cases.